### PR TITLE
Rules page

### PR DESCRIPTION
**Pull request by i-am-a-codegirl**
Please review the rules page of my quiz site

It has the following features

nav : links to categories and rules page
main: Contains the rules along with a call to action to the categories page
Footer: links to the social websites

image

image

### CODE REVIEW

File: rules/rules.css [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+    font-family: 'Inter', sans-serif;
+    font-family: 'Mukta', sans-serif;
+    background-image: url("../images/blue-copy-space-digital-background_23-2148821698.webp");
+    background-repeat: no-repeat;
+    background-size: cover;
+   
+box-sizing: border-box;
+margin: 0px;
+padding: 0px;
+/* background-color: #e056fd;
```
Comment (pnchinmay) [inline rules/rules.css]: Remove these comments

File: rules/rules.css [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+nav h2{
+    position: relative;
+  
+    font-size: 4rem;
+    line-height: 1.2rem;
+    letter-spacing: 0.125rem;
+}
+
+.imp-text{
+    font-family: 'Inter', sans-serif;
```
Comment (pnchinmay) [inline rules/rules.css]: Consider making these indentations uniform

**Review by pnchinmay — APPROVED** [review]
Left a few comments
Everything else looks good :100:

File: rules/rules.css [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+.current-pg{
+    color: var(--secondary-color);
+}
+
+
+
+nav ul{
+    margin-left: auto; 
+    display: flex;
+  gap: 15px;
```
Comment (khushijohri2001) [inline rules/rules.css:78]: Keep the indentation same everywhere. Ideally, 2 spaces in HTML/CSS according to Google Style Guide

**Review by khushijohri2001 — APPROVED** [review]
Remove extra unwanted spaces and align the code with uniform indentation. Otherwise everything is fine

**Pull request merged**